### PR TITLE
EWPP-3873: Update tests for "Consultation" content type.

### PR DESCRIPTION
**Pull request by fmmribeiro**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: tests/src/Functional/ContentConsultationRenderTest.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ protected function setUp(): void {
    * Tests Consultation full view mode rendering.
    */
   public function testConsultationRendering(): void {
-    $this->markTestSkipped('Must be re-enabled before considering migration to ECL 4 as complete.');
+
```
Comment (22Alexandra) [inline tests/src/Functional/ContentConsultationRenderTest.php]: This extra space can be removed.

File: tests/src/Functional/ContentRenderTestBase.php [hunk 2]
```diff
@@ -17,6 +17,7 @@
 use Drupal\oe_content_sub_entity_document_reference\Entity\DocumentReference;
 use Drupal\Tests\BrowserTestBase;
 use Drupal\Tests\oe_theme\PatternAssertions\FieldListAssert;
+use Drupal\Tests\oe_theme\PatternAssertions\SocialMediaLinksAssert;
```
Comment (22Alexandra) [inline tests/src/Functional/ContentRenderTestBase.php:20]: Doesn't seem to be used, should be removed.

File: tests/src/Kernel/ConsultationRenderTest.php [hunk 3]
```diff
@@ -61,7 +61,7 @@ protected function setUp(): void {
    * Test a consultation being rendered as a teaser.
    */
   public function testTeaser(): void {
-    $this->markTestSkipped('Must be re-enabled before considering migration to ECL 4 as complete.');
+
```
Comment (22Alexandra) [inline tests/src/Kernel/ConsultationRenderTest.php]: This extra space can be removed.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

File: tests/src/Functional/ContentRenderTestBase.php (hunk 2 above)
Comment (22Alexandra) [reply tests/src/Functional/ContentRenderTestBase.php:20]: Nvm, it's used in here `assertContactDetailsRender()`.

**Review by 22Alexandra — APPROVED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by 22Alexandra — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**